content/_includes/page/intro.njk

```diff
@@ -1,7 +1,24 @@
+{%- import 'embed.macros.njk' as embed -%}
```

Comment:
Why the `{%-` instead of `{%`?

Reply:
I just copy/pasted in this case, didn't think about it.